app/commands/user/challenges/featured_exercises_progress_48_in_24.rb

```diff
@@ -16,8 +16,8 @@ class User::Challenges::FeaturedExercisesProgress48In24
     { week: 9, slug: 'parallel-letter-frequency', title: 'Parallel Letter Frequency', featured_tracks: %w[go java elixir], learning_opportunity: "Learn about the difference between concurrency and parallellism and how they affect performance. Explore different ways to run code in parallel and how to avoid concurrency issues through mutexes or concurrent collections." },
     { week: 10, slug: 'allergies', title: 'Allergies', featured_tracks: %w[nim elm rust], learning_opportunity: "Learn about bitwise operations, bitshifting, bitwise comparison and other binary number concepts, or try solving the exercise functionally with pattern matching and recursion." },
     { week: 11, slug: 'sieve', title: 'Sieve', featured_tracks: %w[zig bash fortran], learning_opportunity: "Though usually implemented using a fixed-size buffer and nested loops, this exercise is the perfect place to explore bitwise representations and clever loop limits to improve memory and performance." },
-    { week: 12, slug: 'luhn', title: 'Luhn', featured_tracks: %w[typescript perl5 awk], learning_opportunity: "Explore different ways to validate input and to double every other digit, which can be done is some surprising ways, with imperative, OOP and functional ways all possible!" },
-    { week: 13, slug: 'scrabble-score', title: 'Scrabble Score', featured_tracks: %w[python scheme c], learning_opportunity: "" },
+    { week: 12, slug: 'luhn', title: 'Luhn', featured_tracks: %w[typescript perl5 awk], learning_opportunity: "Explore different ways to validate input and to double every other digit, which can be done in some surprising ways, with imperative, OOP and functional ways all possible!" },
+    { week: 13, slug: 'scrabble-score', title: 'Scrabble Score', featured_tracks: %w[python scheme c], learning_opportunity: "Learn to use loops, dictionaries, pattern matching and lookup tables to sum a word's letter scores! You'll also see two very creative regular expression solutions." },
```

Comment:
These one's aren't about the videos themselves - just the exercise. So maybe:

```suggestion
    { week: 13, slug: 'scrabble-score', title: 'Scrabble Score', featured_tracks: %w[python scheme c], learning_opportunity: "Learn to use loops, dictionaries, pattern matching and lookup tables to sum a word's letter scores! Or try and get creative to solve it with regular expressions." },
```